source/common/network/common_connection_filter_states.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "envoy/common/execution_context.h"
+#include "envoy/network/connection.h"
+#include "envoy/stream_info/filter_state.h"
+
+namespace Envoy {
+
+static constexpr absl::string_view kConnectionExecutionContextFilterStateName =
+    "connection-execution-context";
```

Comment:
Because this will be a feature that in the core case base. So it would be better to ensure the name style is consistent with current well known filter state names'.

See the https://www.envoyproxy.io/docs/envoy/latest/configuration/advanced/well_known_filter_state#well-known-filter-state.

Reply:
Done. I did not know that, thanks for pointing it out.